packages/rocq-prover/rocq-prover.9.0.0/opam

```diff
@@ -0,0 +1,54 @@
+opam-version: "2.0"
+synopsis: "The Rocq Prover with Stdlib"
+description: """
+The Rocq Prover is an interactive theorem prover, or proof assistant. It provides
+a formal language to write mathematical definitions, executable
+algorithms and theorems together with an environment for
+semi-interactive development of machine-checked proofs.
+
+Typical applications include the certification of properties of
+programming languages (e.g. the CompCert compiler certification
+project, or the Bedrock verified low-level programming library), the
+formalization of mathematics (e.g. the full formalization of the
+Feit-Thompson theorem or homotopy type theory) and teaching.
+
+This package is a virtual package gathering the rocq-core and rocq-stdlib packages."""
```

Comment:
This probably shouldn't have any build nor src then?

Reply:
Indeed, fixed as well.

Reply:
The `(allow-empty)` in the repo's dune-project does not seem to affect this correctly...

Reply:
I guess we'll have to tweak the *.opam.template files once more.